### DISCUSSION TITLE
Feature request: QTime alternative representation

### DISCUSSION THREAD

**Discussion by hendrik-schneider** [quasarframework/quasar#14044] [Ideas / Proposals]
Hello Quasar-Team👋,

we have a lot of business customers working with our application and there is a large consistency among our users that they do not understand how to deal with the `Qtime`-representation (An analog clock for the hour, for the minute and at least for the second). They need time to understand the input every time they have to do it. [quasar.dev/vue-components/time](https://quasar.dev/vue-components/time) (I am developer, I have no problems with it, but I want the best usability 😉)

I dont want to use a third component for this and i  also dont want to use the native representation, cause its not in the "quasar"-style and it looks and feels different on every device, so i hope to "get the ball rolling" for an alternative representation for the `QTime`

**_What our users expect:_**
They expect `time picker wheels` side by side as you know it for example from IOS - with the possibility to roll up and down with the finger.

**Do you have similar experience or feedback?**
**Is there a possibility to consider and discuss this request for Quasar 3?**

Thank you for any kind of participation 🙋‍♂️

**Comment by namespace-github** [comment]
I have replaced QTime with the native picker, because we also had problems with this variant. This is not pretty, because the native picker looks different on each device and can not be changed via CSS. 

In my opinion the time picker wheels are the better known and more common variant. So I join the request. Go for it 👍

**Comment by Bananamoon** [comment]
Although an old feature request, I also (still) vouch for this! Maybe do it with an option to choose between analog/digital representation of the time.
I don't find the current clock very user friendly and am now using another library for this. Unfortunately this means I have differences in style, animation and behavior.
Please please implement this!
